lib/preparermd/plugins/metadata_envelopes.rb

```diff
@@ -38,12 +38,11 @@ def render_json(page, site)
 
       output = page.to_liquid
 
-      envelope = {
-        title: output["title"],
-        body: output["content"],
-        layout_key: layout,
-        categories: output["categories"] || []
-      }
+      envelope = page.data;
```

Comment:
1. No semicolons in Ruby :wink:
2. I'm wondering if this should be `page.data.dup` so we don't modify the original data structure in unexpected ways? If you were able to run this locally without issues, it's probably fine.